src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+
+import duke.exceptions.*;
```

Comment:
According to the Java code standard, it is a good practice that imported classes should always be listed explicitly.